build.cake

```diff
@@ -0,0 +1,69 @@
+#tool "nuget:?package=JetBrains.ReSharper.CommandLineTools"
+#tool "nuget:?package=NVika.MSBuild"
+var NVikaToolPath = GetFiles("./tools/NVika.MSBuild.*/tools/NVika.exe").First();
+var CodeFileSanityToolPath = DownloadFile("https://github.com/peppy/CodeFileSanity/releases/download/v0.2.5/CodeFileSanity.exe");
+
+///////////////////////////////////////////////////////////////////////////////
+// ARGUMENTS
+///////////////////////////////////////////////////////////////////////////////
+
+var target = Argument("target", "Build");
+var framework = Argument("framework", "net471");
```

Comment:
Desktop has netcoreapp2.1 as target framework

Reply:
This is true, but it was always build with net471 in the appveyor.xml. @peppy can we switch over to netcoreapp2.1 or do we stay on net471 for now?

Reply:
appveyor use value from csproj -> netcoreapp2.1

Reply:
nevermind then

Reply:
We made the complete switch yesterday via https://github.com/ppy/osu/pull/3061